### DISCUSSION TITLE
course/starbook/register-general

### DISCUSSION THREAD

**Discussion by giscus** [gogf/gf-site#68] [General]
# course/starbook/register-general

使用GoFrame框架开发用户注册接口，包含添加API、编写业务逻辑层、控制器调用逻辑、控制器注册以及项目运行的详细步骤。通过生成的数据访问对象和数据模型，实现接口与数据库的交互，最终在项目运行中测试接口的功能。

https://goframe.org/course/starbook/register-general

**Comment by Mobs-demacia** [comment]
gf run .\main.go
build: .\main.go
go build -o .\main.exe  .\main.go
.\main.exe 
build running pid: 3268
2024-11-19 15:12:24.948 [FATA] {384712c4224d091843e3837627848d3b} invalid pattern "/register@default", URI should lead with '/' 
Stack:
1.  gf-admin/internal/cmd.init.func1
    D:/code/gf-admin/internal/cmd/cmd.go:26
2.  main.main
    D:/code/gf-admin/main.go:14

跟着代码一点一点敲的 会报错

**Reply by Mobs-demacia** [reply]
大家别学我, 要多看文档, 路由在注册的时候都是以/开头的 我就是在v1那里少写了/
